otcextensions/tests/functional/osclient/nat/v2/test_nat_gateway.py

```diff
@@ -0,0 +1,104 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+
+from otcextensions.tests.functional.osclient.nat.v2 import common
+
+
+class TestNatGateway(common.NatTestCase):
+    """Functional Tests for NAT Gateway"""
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
this will be failing after #73 is merged - drop it please

Reply:
Update to setUp